test_activate_operation_updates_active_users needs a review

I see this sometimes on my local tests
```
[gw5] [100%] PASSED tests/_test_msui/test_mscolab.py::Test_Mscolab::test_activate_operation_updates_active_users 

Exception ignored in: <function WMSControlWidget.__del__ at 0x718a67ac9120>
Traceback (most recent call last):
  File "/home/reimar/MAIN/MSS/mslib/msui/wms_control.py", line 550, in __del__
    self.service_cache()
  File "/home/reimar/MAIN/MSS/mslib/msui/wms_control.py", line 1455, in service_cache
    files = [os.path.join(self.wms_cache, f) for f in os.listdir(self.wms_cache)]
                                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] Datei oder Verzeichnis nicht gefunden: '/tmp/tmpnodr4uwv/msui/wms_cache/'
```

We have to set wms_cache
https://github.com/Open-MSS/MSS/blob/stable/tests/_test_msui/test_wms_control.py#L500

similiar as here
https://github.com/Open-MSS/MSS/blob/stable/tests/_test_msui/test_wms_control.py#L69

On the MBP I got today
```
Failed: An unhandled message box popped up during your test!
E       PyQt5.QtWidgets.QMessageBox.critical(): [call(<mslib.msui.wms_control.HSecWMSControlWidget object at 0x34be62950>, 'Web Map Service', "ERROR: Cannot delete file '/Users/reimarbauer/Library/Caches/msui/wms_cache/6a69a349092204e79ecfb3fabe12998a.png'. (<class 'FileNotFoundError'>: [Errno 2] No such file or directory: '/Users/reimarbauer/Library/Caches/msui/wms_cache/6a69a349092204e79ecfb3fabe12998a.png')")]
```

that looks like a test does not use a tmp dir. It is not clear yet if it is the same problem.

--- a/tests/_test_msui/test_wms_control.py
+++ b/tests/_test_msui/test_wms_control.py
@@ -496,8 +496,12 @@ class TestWMSControlWidgetSetupSimple:
 
     @pytest.fixture(autouse=True)
     def setup(self, qtbot):
+        wc.WMS_SERVICE_CACHE = {}
         self.view = HSecViewMockup()
-        self.window = wc.HSecWMSControlWidget(view=self.view)
+        self.tempdir = tempfile.mkdtemp()
+        if not os.path.exists(self.tempdir):
+            os.mkdir(self.tempdir)
+        self.window = wc.HSecWMSControlWidget(view=self.view, wms_cache=self.tempdir)
         self.window.show()
 
         # Remove all previous cached URLs
@@ -507,6 +511,10 @@ class TestWMSControlWidgetSetupSimple:
 
         yield
         self.window.hide()
+
+    def test_wms_cache_tmp(self):
+        assert self.window.wms_cache is not None
+        assert "tmp" in self.window.wms_cache
 
     def test_xml(self):
         testxml = self.xml.format("", self.srs_base, self.dimext_time + self.dimext_inittime + self.dimext_elevation)

--- a/tests/_test_msui/test_wms_control.py
+++ b/tests/_test_msui/test_wms_control.py
@@ -495,10 +495,10 @@ class TestWMSControlWidgetSetupSimple:
         <Extent name="ELEVATION" default="900.0"> 500.0,600.0,700.0,900.0 </Extent>"""
 
     @pytest.fixture(autouse=True)
-    def setup(self, tmp_path_factory, qtbot):
+    def setup(self, tmp_path, qtbot):
         wc.WMS_SERVICE_CACHE = {}
         self.view = HSecViewMockup()
-        self.tempdir = tmp_path_factory.mktemp("TMP_WMS")
+        self.tempdir = tmp_path
         self.window = wc.HSecWMSControlWidget(view=self.view, wms_cache=self.tempdir)
         self.window.show()
 

--- a/tests/_test_msui/test_wms_control.py
+++ b/tests/_test_msui/test_wms_control.py
@@ -512,7 +512,8 @@ class TestWMSControlWidgetSetupSimple:
 
     def test_wms_cache_tmp(self):
         assert self.window.wms_cache is not None
-        assert "TMP_WMS" in self.window.wms_cache
+        # wms_cache has an ending path separator
+        assert str(self.tempdir) == self.window.wms_cache[:-1]
 
     def test_xml(self):
         testxml = self.xml.format("", self.srs_base, self.dimext_time + self.dimext_inittime + self.dimext_elevation)

--- a/tests/_test_msui/test_wms_control.py
+++ b/tests/_test_msui/test_wms_control.py
@@ -495,12 +495,10 @@ class TestWMSControlWidgetSetupSimple:
         <Extent name="ELEVATION" default="900.0"> 500.0,600.0,700.0,900.0 </Extent>"""
 
     @pytest.fixture(autouse=True)
-    def setup(self, qtbot):
+    def setup(self, tmp_path_factory, qtbot):
         wc.WMS_SERVICE_CACHE = {}
         self.view = HSecViewMockup()
-        self.tempdir = tempfile.mkdtemp()
-        if not os.path.exists(self.tempdir):
-            os.mkdir(self.tempdir)
+        self.tempdir = tmp_path_factory.mktemp("TMP_WMS")
         self.window = wc.HSecWMSControlWidget(view=self.view, wms_cache=self.tempdir)
         self.window.show()
 
@@ -514,7 +512,7 @@ class TestWMSControlWidgetSetupSimple:
 
     def test_wms_cache_tmp(self):
         assert self.window.wms_cache is not None
-        assert "tmp" in self.window.wms_cache
+        assert "TMP_WMS" in self.window.wms_cache
 
     def test_xml(self):
         testxml = self.xml.format("", self.srs_base, self.dimext_time + self.dimext_inittime + self.dimext_elevation)

